templates/base.html.twig

```diff
@@ -58,6 +58,15 @@
             </nav>
         </footer>
 
+        <script>
+            jQuery(document).on('click', '[data-toggle="lightbox"]', function(event) {
+                event.preventDefault();
+                jQuery(this).ekkoLightbox({
+                    alwaysShowClose: true
+                });
+            });
+        </script>
+
```

Comment:
Shouldn't this be in view..html.twig? Otherwise can we move it into foot.html.twig?

Reply:
moved